### PR TITLE
Fixed the broken components and formatted the IP insights notebook

### PR DESCRIPTION
**Pull request by vivekmadan2**
Fixed and formatted IP insights notebook.

@reviewer: **Please do not merge**
1. Waiting for legal approval regarding hosting dataset on S3 bucket.
2. Name of the data bucket would change to jump-start and needs to be reflected in this PR.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: introduction_to_amazon_algorithms/ipinsights_login/ipinsights-tutorial.ipynb [hunk 1]
```diff
@@ -52,25 +52,25 @@
     "\n",
```
Comment (yastasho) [inline introduction_to_amazon_algorithms/ipinsights_login/ipinsights-tutorial.ipynb:52]: <p>Minor: line 17 is under-indented.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1711/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (yastasho) [inline introduction_to_amazon_algorithms/ipinsights_login/ipinsights-tutorial.ipynb:52]: <p>Can  <code>num_flagged_cases</code> or <code>num_all_positives</code> be zero? If yes, the code can fail.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1711/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Comment (yastasho) [inline introduction_to_amazon_algorithms/ipinsights_login/ipinsights-tutorial.ipynb:52]: <p>Formatting: inconsistent usage of space characters after <code>,</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1711/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Comment by yastasho** [comment]
> @yastasho: I did that exercise: copy the cells (except one of them) to pycharm and uses Black to format. Could please point to one or two "minor formatting issues throughout the notebook" so that I have an idea.
> 
> Will address the other comments.

Please check comments prefixed with "Formatting:". Those are minor things though.

File: introduction_to_amazon_algorithms/ipinsights_login/ipinsights-tutorial.ipynb (hunk 1 above)
Comment (vivekmadan2) [reply introduction_to_amazon_algorithms/ipinsights_login/ipinsights-tutorial.ipynb:52]: <p>fixed.</p>

Comment (vivekmadan2) [reply introduction_to_amazon_algorithms/ipinsights_login/ipinsights-tutorial.ipynb:52]: <p>Not sure what you meant here.</p>

**Review by mabunday — DISMISSED** [review]

**Review by mabunday — DISMISSED** [review]

**Review by aaronmarkham — APPROVED** [review]

**Pull request merged**